docs/api/web-contents.md

```diff
@@ -588,6 +588,15 @@ Emitted when media starts playing.
 
 Emitted when media is paused or done playing.
 
+#### Event: 'audio-state-changed'
+
+Returns:
+
+* `event` Event
+* `audible` boolean - Whether audio is currently playing.
```

Comment:
Yeah, I do think this is a good target for using the new style of event! It seems unlikely that this event will grow extra properties, but you never now...
```suggestion
* `event` Event<>
  * `audible` boolean - Whether audio is currently playing.
```